lib/bulk_processor.rb

```diff
@@ -1,3 +1,6 @@
+require 'bulk_processor/back_end'
+require 'bulk_processor/back_end/active_job'
+require 'bulk_processor/back_end/dynosaur'
```

Comment:
Is there a straightforward way to only load Dynosaur if you configure that backend, and similarly with ActiveJob`?